src/test/java/hudson/plugins/ansicolor/AnsiColorBuildWrapperTest.java

```diff
@@ -42,17 +45,21 @@ public void testDecorateLogger() {
     public static BuildWatcher buildWatcher = new BuildWatcher();
     @Rule
     public RestartableJenkinsRule story = new RestartableJenkinsRule();
+    @Rule
+    public LoggerRule logging = new LoggerRule().recordPackage(ConsoleNote.class, Level.FINE);
 
+    @Issue("JENKINS-54133")
     @Test
     public void testWorkflowWrap() throws Exception {
         story.addStep(new Statement() {
 
             @Override
             public void evaluate() throws Throwable {
                 Assume.assumeTrue(!Functions.isWindows());
+                story.j.createSlave();
                 WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "p");
                 p.setDefinition(new CpsFlowDefinition(
-                        "node {\n"
+                        "node('!master') {\n"
```

Comment:
Feels like this should be a separate test.

Reply:
It seemed unnecessary as `AnsiColorStepTest` already tests the behavior on master.